numalogic/models/threshold/_median.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import numpy.typing as npt
+from typing_extensions import Self, Final
+
+from numalogic.base import BaseThresholdModel
+from numalogic.tools.exceptions import InvalidDataShapeError, ModelInitializationError
+
+_INLIER: Final[int] = 0
+_OUTLIER: Final[int] = 1
+_INPUT_DIMS: Final[int] = 2
```

Comment:
You think these can be class variables or another ENUM class?

Reply:
I think we should make it a ENUM class because even _std.py can use that.

Reply:
We can do that later. Minor thing.